policy-supplemental/platforms.md

```diff
@@ -0,0 +1,181 @@
+## Current primary platforms
+
+| Target           | &nbsp; | O/S                   | &nbsp; | Architecture | &nbsp; | Toolchain                            |
+|------------------|--------|-----------------------|--------|--------------|--------|--------------------------------------|
+| linux-x86\_64    |        | Ubuntu Server 20.04.3 |        | x86\_64      |        | gcc 9.3.0                            |
```

Comment:
A thought: could these tables be sorted?

Reply:
Other PR, I want this to be an easy to verify straight copy of the file in openssl/web.git